_content/tour/eng/maps.article

```diff
@@ -27,6 +27,8 @@ block of memory underneath.
 .play maps/example5.go
 .play maps/example6.go
 .play maps/example7.go
+.play maps/example8.go
```

Comment:
You forgot to list these two new examples above.

Reply:
Thanks
Done